### PR TITLE
[13.0][MIG] procurement_mto_analytic

### PR DESCRIPTION
**Pull request by ps-tubtim**
Standard migration.

### CODE REVIEW

**Comment by sa3m** [comment]
How can we help merge this PR please ?

**Comment by AaronHForgeFlow** [comment]
you can test the functionality on runbot or do a code review. So far there is not any review.

**Review by rousseldenis — APPROVED** [review]

**Comment by sa3m** [comment]
It seems that the runbot instance is broken, the sale module was not installed, analytic was installed but not accessible.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional test on local

File: procurement_mto_analytic/README.rst [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to your user settings.
+#. Enable *Analytic Accounting for Purchases* in *Technical Settings*.
```
Comment (AaronHForgeFlow) [inline procurement_mto_analytic/README.rst]: It seems this groups does not exists now.

**Pull request merged**